packages/mobile/src/screens/track-screen/TrackScreenDetailsTile.tsx

```diff
@@ -532,7 +532,7 @@ export const TrackScreenDetailsTile = ({
       headerText={isRemix ? messages.remix : messages.track}
       hideFavorite={hideFavorite}
       hideRepost={hideRepost}
-      hideShare={is_unlisted && !field_visibility?.share}
+      hideShare={!isOwner || !field_visibility?.share}
```

Comment:
This hides share in all cases if you aren't the owner?

Reply:
oh right... that's not what we want.